maven/jetifier.bzl

```diff
@@ -55,7 +55,8 @@ def jetifier_init():
 
 # jetify_jar based on https://github.com/bazelbuild/tools_android/pull/5
 def jetify_jar(ctx, jar):
-    jetified_outfile = ctx.actions.declare_file("%s-jetified.%s" % (ctx.attr.name, jar.extension))
+    basename = jar.basename.rsplit(".", 1)[0]
```

Comment:
question... how come basename doesn't give basename? =)

Reply:
/foo/bar.baz -> basename -> bar.baz

Unix basename lets you ALSO add a suffix to strip off, but starlark's doesn't.